ecommerce/regis/static/sass/base/_receipt.scss

```diff
@@ -0,0 +1,206 @@
+// Receipt styles ecommerce/ecommerce/static/sass/partials/views/_receipt.scss.
+
+.receipt {
+  padding-top: 45px;
+
+  h2 {
+    color: initial;
+    margin-bottom: 30px;
+  }
+
+  .order-headers {
+    color: initial;
```

Comment:
```suggestion
    color: initial;
    margin-top: 10px;    
```

Reply:
This change is not necessary, there is already a margin-top definition for that selector. @kuipumu